graphql-server/pkg/resolvers/proposal.go

```diff
@@ -88,6 +90,61 @@ func (r *proposalResolver) MyReaction(ctx context.Context, obj *models.Proposal)
 	return &reaction.Reaction, nil
 }
 
+func (r *proposalResolver) Turnout(ctx context.Context, obj *models.Proposal) (*string, error) {
+	stakingPool, err := pkgContext.GetQueriesFromCtx(ctx).Proposal.QueryProposalStakingPool(obj.ID)
+	if stakingPool == nil {
+		return nil, nil
+	}
+	if err != nil {
+		return nil, err
+	}
+	bondedTokens, err := bunbig.NewFloat().FromString(stakingPool.BondedTokens.String())
+	if err != nil {
+		return nil, err
+	}
+	tallyResult, err := r.TallyResult(ctx, obj)
+	if err != nil {
+		return nil, err
+	}
+
+	voted := tallyResult.Yes.Add(tallyResult.No).Add(tallyResult.NoWithVeto).Add(tallyResult.Abstain)
+	votedFloat, err := bunbig.NewFloat().FromString(voted.String())
+	if err != nil {
+		return nil, err
+	}
+
+	turnout := votedFloat.Div(bondedTokens).String()
+	return &turnout, nil
+}
+
+func (r *proposalResolver) MyVote(ctx context.Context, obj *models.Proposal) (*models.ProposalVote, error) {
+	address, err := utils.GetOperationArgumentValue(ctx, "input.address.address")
+	if err != nil {
+		return nil, nil
+	}
+
+	key := dataloaders.ProposalVoteKey{ProposalID: obj.ID, Address: address}
+	vote, err := pkgContext.GetDataLoadersFromCtx(ctx).Proposal.LoadProposalVote(key)
+	if err != nil {
+		return nil, err
+	}
+	return vote, nil
+}
+
+func (r *proposalResolver) MyDeposit(ctx context.Context, obj *models.Proposal) (*models.ProposalDeposit, error) {
+	address, err := utils.GetOperationArgumentValue(ctx, "input.address.address")
```

Comment:
what if we introduce another query that returns Proposal type in the future? Assuming the parent operation here makes this un-reusable in the future

can we get the user address from request header / cookie?

either: a) require all user to auth on wallet connection, or b) send unauthenticated user address for query use in header?